### DISCUSSION TITLE
Quick question about translations.

### DISCUSSION THREAD

**Discussion by PhiMyth-zz** [Revolutionary-Games/Thrive#2163] [Q&A] [answered]
Hello. I've got a quick question about translations.

Every how many days/weeks do Weblate translations get merged?
This way I can plan things out, because I like planning things.
I placed it here, because I couldn't find a more suitable location other than this one.

Thank you.
Phi

**Comment by hhyyrylainen** [comment, marked as answer]
I've been merging them about once a week. Lately I've been making about one devbuild per week and for that I've merged the translations before that. If I don't merge after 8-9 days Weblate seems to email me.

**Reply by PhiMyth-zz** [reply]
Thank you for letting me know! 👍🏼
